### PR TITLE
Add Oct meeting minutes

### PR DESCRIPTION
**Pull request by csutherl**

### CODE REVIEW

**Review by arecvlohe — DISMISSED** [review]

File: minutes/board_2020_10_08.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+*   Adam
+*   Nick
+*   Coty
+
+### Action Items from Previous Meeting
+
+*   Adam
+    *   ~~Create a shared team file on keybase to share passwords (or some other way) Ended up using 1password~~
+    *   Finalize CFP for Natives in Tech Conf 2020 and publicize it
+    *   Setup a meeting for end of Sept to have a strategy meeting about the overall goals of NiT
```
Comment (erinishimoticha) [inline minutes/board_2020_10_08.md]: Should be something like "Set up a strategy meeting for end of Sept to talk about the overall goals of NiT"

File: minutes/board_2020_10_08.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        *   See
+            [*https://github.com/nativesintech/nonprofit-governance/pull/8*](https://github.com/nativesintech/nonprofit-governance/pull/8)
+        *   I think we're going to abandon that because it's too
+            difficult to collaborate on GitHub.
+
+### Updates
+
+*   Adam
+    *   Reaching out to people to help with NiT Conf 2020
+        *   Thinking about having it pre*recorded as opposed to live
```
Comment (erinishimoticha) [inline minutes/board_2020_10_08.md]: "pre-recorded"

File: minutes/board_2020_10_08.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        *   Goal is to get this stuff out by EOM
+        *   Reach out to speakers directly
+    *   Collaborating with Eli and Jay on rebrand of NiT site
+        *   Working through designs and feedback
+        *   Eli might write an article about what inspired the rebrand
+    *   Looked into trademarking NiT new logo
+        *   Lot's of lawyer speak
```
Comment (erinishimoticha) [inline minutes/board_2020_10_08.md]: "Lots"

**Review by erinishimoticha — DISMISSED** [review]
Couple of tiny updates to do.

**Comment by csutherl** [comment]
The base branch requires all commits to be signed, so I had to sign the commit and push again. Nothing changed other than the signature.

**Comment by csutherl** [comment]
Corrected the issues @erinishimoticha noted above.

File: minutes/board_2020_10_08.md (hunk 3 above)
Comment (csutherl) [reply minutes/board_2020_10_08.md]: Fixed.

File: minutes/board_2020_10_08.md (hunk 2 above)
Comment (csutherl) [reply minutes/board_2020_10_08.md]: Fixed.

File: minutes/board_2020_10_08.md (hunk 1 above)
Comment (csutherl) [reply minutes/board_2020_10_08.md]: Fixed.

**Review by arecvlohe — APPROVED** [review]

**Review by microfauna — APPROVED** [review]

**Pull request merged**